src/setup/cortx_deploy

```diff
@@ -71,6 +75,9 @@ while [ $# -gt 0 ];  do
         shift 1
         CORTX_CONFSTORE=$1
         ;;
+    -o )
+       OVERRIDE="-o"
```

Comment:
Can we call it "FORCE_DEPLOY" to make it more intuitive.

Reply:
Done.